### ISSUE TITLE
No TTL on A or CNAME records

### ISSUE THREAD

**Issue by markeganfuller** [infobloxopen/infoblox-client#193]
TTL can neither be queried nor set on A or CNAME records.

* If a A or CNAME is created with a TTL, the record is created but the TTL is not set.
* If a record with a TTL set is queried, there is no way to access this value.

e.g.

```python
# Results in A record without TTL
objects.ARecord.create(
    connector=conn,
    ip='127.1.1.1',
    name="an.example.com",
    ttl=300,
    view="Internal"
)
```

```python
# A record with TTL set via web interface
example = objects.ARecord.search(connector=conn, ipv4addr="127.1.1.1")
example.ttl  # Attribute Error
example.__dict__

{'connector': <infoblox_client.connector.Connector at 0x7fe494811fd0>,
 'ipv4addr': '127.1.1.1',
 'name': 'an.example.com',
 'view': 'Internal',
 'extattrs': None,
 '_ref': 'record:a/ZG5zLmJpbmRfYSQuMS51ay5hYy5xbXVsLmhwYyx0ZXN0LDEyNy4xLjEuMQ:an.example.com/Internal'
}
```

**Comment by markeganfuller** [comment]
This can be worked around with the low level API:

```python
r = conn.get_object(
    'record:a',
    {
        'view': 'Internal',
        'name': 'an.example.com',
    }
)

conn.update_object(
    r[0]['_ref'],
    {
        'ttl': 300
    }
)
```

**Comment by saiprasannasastry** [comment]
![image](https://user-images.githubusercontent.com/36291746/63001770-f74c0300-be91-11e9-975c-cdde0b709e8d.png)
cant search ttl

**Comment by saiprasannasastry** [comment]
PR has been raised and it will be merged by today or tomorrow. Closing this as the feature has been added, how to return is addressed in #194
